boards/arm/lpcxpresso54114_m0/board.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2017, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __INC_BOARD_H
+#define __INC_BOARD_H
+
+#include <soc.h>
+
+/* Red LED */
+#define RED_GPIO_NAME	CONFIG_GPIO_MCUX_LPC_PORT0_NAME
```

Comment:
Would be nice if this moved to being in dts.

Reply:
Blinky demo no longer builds even for master core on master branch.
I have put moved it into DTS but commented it there. Need to add missing gpio sections in soc DTS.
I have tried it but hasn't been able to make it work.
So blinky demo still does not work.

Reply:
leds are fixed and in dts now